content/doc/tutorials/build-a-labview-app.adoc

```diff
@@ -0,0 +1,384 @@
+
+. Enter a descriptive name for the token, such as ‘Jenkins LabVIEW Token’, and select the _repo_ scope. 
+[.boxshadow]
+image:tutorials/labview-11-generate-access-token.png[alt="generate access token",width=100%]
+
+. At the bottom of the page, click *Generate token*.
+
+. Copy down your Personal access token. This is your only chance to copy the token. Keep your access token safe, so others can’t access your GitHub. 
+[.boxshadow]
+image:tutorials/labview-12-post-access-token-gen.png[alt="post access token",width=100%]
+
+Congrats! You’ve set up the GitHub repositories you’ll need for this example, and you have the personal access token so Jenkins can access your GitHub information. Next, we’ll configure Jenkins to use these items. 
+
+=== Jenkins Setup
+
+==== Jenkins Initial Setup
+
+*Note*: This tutorial uses the classic Jenkins view, with minimal plugins to get the example running. You can use additional plugins such as https://jenkins.io/doc/book/blueocean/[Blue Ocean] for an enhanced UI experience.
+
+. Install Jenkins based on your Operating System using the https://jenkins.io/doc/book/installing/[help], and navigate to the address of your Jenkins server in your browser. A few additional notes (all of which are already mentioned in the aforementioned Jenkins help):
+** The default Jenkins address is http://localhost:8080 
+** The first time you launch Jenkins, you will need to unlock it with the admin password from the Console log output
+[.boxshadow]
+image:tutorials/labview-13-unlocking-jenkins.png[alt="unlocking jenkins",width=100%]
+
+. When prompted to Customize Jenkins, select the *Select plugins to install* option. Here, make sure the following Plugins are selected:
+** GitHub Branch Source
+** Folders Plugin
+** Pipeline: GitHub Groovy Libraries
+
+. Click *Install* at the bottom of the page to install the plugins.
+
+. Once plugins are complete, you’ll be prompted to create a user. Enter your credentials and click Save and Continue.
+[.boxshadow]
+image:tutorials/labview-14-create-first-user.png[alt="create first user",width=100%]
+. On the following Instance Configuration page, leave the Jenkins URL as-is.
+. Click *Start using Jenkins* to begin your Jenkins configuration.
+
+==== Setting up Environment Variables
+
+Environment variables can be accessed across Jenkins jobs. We will want certain values accessible when Jenkins tries to build, test and diff your application. Since the GitHub organization name, access token, and picture repository will likely be the same across Jenkins jobs, we will set them in our Jenkins configuration.
+
+. From the Jenkins dashboard, navigate to the system configuration page: *Manage Jenkins >> Configure System*
+. Navigate to the _Global properties_ section.
+. Check the *Environment* variables box to display the List of variables.
+. Click *Add* to add a new environment variable. Fill out the Name as shown below, and enter your GitHub access token in the Value field:
```

Comment:
That makes sense to me, thanks for the feedback! We'll work on this.

Reply:
To confirm, it looks like instead of using our GitHub Access Token as an environment variable, the better option would be setting it as [Secret Text](https://jenkins.io/doc/book/using/using-credentials/), and referencing it in our Jenkinsfile/groovy files [similar to here](https://jenkins.io/doc/book/pipeline/jenkinsfile#secret-text):

`AWS_ACCESS_KEY_ID = credentials('jenkins-aws-secret-key-id')`

Reply:
I updated the tutorial (and [relevant files here](https://forums.ni.com/t5/Example-Program-Drafts/LabVIEW-Continuous-Integration-Jenkins-GitHub/ta-p/3960832)) to use a 'Secret text' credential for the GitHub access token. 

The access token is referenced with the below method (in `lvPipeline.groovy` where the files are hosted):
```
withCredentials([string(credentialsId: 'github-access-token', variable: 'accessToken')]) {
      //GitHub-relevant actions here
}
```